### ISSUE TITLE
Could not extract stream for this track: TypeError: value.matchAll is not a function or its return value is not iterable

### ISSUE THREAD

**Issue by XCraftTM** [zyrouge/node-youtube-ext#16]
So i just updated to the newest version of youtube-ext and it showed me this error, any idea why that could happen?
```js
Could not extract stream for this track

TypeError: value.matchAll is not a function or its return value is not iterable
    at parseYoutubeKeywords (C:\Users\XXX\XXX\TestBot2\node_modules\youtube-ext\dist\utils\youtube.js:31:27)
    at C:\Users\XXX\XXX\TestBot2\node_modules\youtube-ext\dist\videoInfo.js:139:52
    at Generator.next (<anonymous>)
    at fulfilled (C:\Users\XXX\XXX\TestBot2\node_modules\youtube-ext\dist\videoInfo.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

I am using it in combination with discord-player.
youtube-ext: "1.1.15"

**Comment by XCraftTM** [comment]
Adding to this Issue, downgrading to "1.1.14" it works just fine...

**Comment by XCraftTM** [comment]
Fixed with 1.1.16
